lua/packer.lua

```diff
@@ -34,7 +34,8 @@ local config_defaults = {
       current_branch = 'rev-parse --abbrev-ref HEAD',
       diff = 'log --color=never --pretty=format:FMT --no-show-signature HEAD@{1}...HEAD',
       diff_fmt = '%%h %%s (%%cr)',
-      git_diff_fmt = 'show --no-color --pretty=medium %s',
+      -- TODO maybe --no-pager should be added for all commands?
```

Comment:
Re. this TODO just so it doesn't linger here forever, what would the benefit of doing this be vs the current behaviour? What advantage does it provide?

Reply:
Ah right! I initially didn't get the full diff from the command and I thought it was due do this. But it was actually since only the first entry was used here:
https://github.com/wbthomason/packer.nvim/blob/master/lua/packer/display.lua#L674

Removed --no-pager again and the comment :+1: